modules/integration/src/main/java/webindex/integration/DevServer.java

```diff
@@ -0,0 +1,169 @@
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package webindex.integration;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import com.google.gson.Gson;
+import org.apache.accumulo.minicluster.MiniAccumuloCluster;
+import org.apache.accumulo.minicluster.MiniAccumuloConfig;
+import org.apache.commons.io.FileUtils;
+import org.apache.fluo.api.client.FluoAdmin;
+import org.apache.fluo.api.client.FluoClient;
+import org.apache.fluo.api.client.FluoFactory;
+import org.apache.fluo.api.client.LoaderExecutor;
+import org.apache.fluo.api.config.FluoConfiguration;
+import org.apache.fluo.api.mini.MiniFluo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import webindex.core.IndexClient;
+import webindex.core.models.Page;
+import webindex.data.fluo.PageLoader;
+import webindex.data.spark.IndexEnv;
+import webindex.ui.WebServer;
+
+public class DevServer {
+
+  private static final Logger log = LoggerFactory.getLogger(DevServer.class);
+  private static final int TEST_SPLITS = 119;
+
+  private Path dataPath;
+  private Path tempDir;
+  private int webPort;
+  private Path templatePath;
+  private MiniAccumuloCluster cluster;
+  private WebServer webServer;
+  private IndexClient client;
+  private AtomicBoolean started = new AtomicBoolean(false);
+
+  public DevServer(Path dataPath, Path tempDir, int webPort, Path templatePath) {
+    this.dataPath = dataPath;
+    this.tempDir = tempDir;
+    this.webPort = webPort;
+    this.templatePath = templatePath;
+    this.webServer = new WebServer();
+  }
+
+  public IndexClient getIndexClient() {
+    if (!started.get()) {
+      throw new IllegalStateException("DevServer must be started before retrieving index client");
+    }
+    return client;
+  }
+
+  public void start() throws Exception {
+    log.info("Starting WebIndex development server...");
+
+    Path baseDir;
+    if (Files.exists(tempDir) && Files.isDirectory(tempDir)) {
+      baseDir = Paths.get(tempDir.toString(), "webindex-dev-" + UUID.randomUUID());
+    } else {
+      throw new IllegalStateException(
+          "Dev server cannot be started as '/tmp/' directory does not exist");
+    }
+
+    log.info("Starting MiniAccumuloCluster at {}", baseDir);
+    FileUtils.deleteDirectory(baseDir.toFile());
+    MiniAccumuloConfig cfg = new MiniAccumuloConfig(baseDir.toFile(), "secret");
+    cluster = new MiniAccumuloCluster(cfg);
+    cluster.start();
+
+    FluoConfiguration config = new FluoConfiguration();
+    config.setMiniStartAccumulo(false);
+    config.setApplicationName("webindex-dev");
+    config.setAccumuloInstance(cluster.getInstanceName());
+    config.setAccumuloUser("root");
```

Comment:
Can you used the [helper method](https://github.com/apache/incubator-fluo-recipes/blob/f7635dc89219e0cc45376ad318c328dfa199d63d/modules/test/src/main/java/org/apache/fluo/recipes/test/AccumuloExportITBase.java#L223) in Fluo Recipes for this?